apps/align.cpp

```diff
@@ -73,11 +73,13 @@ int main(int argc, char ** argv)
   voxelgrid.filter(*downsampled);
   source_cloud = downsampled;
 
+  pcl::PointCloud<pcl::PointXYZ>::Ptr aligned;
+
   // benchmark
   std::cout << "--- pcl::GICP ---" << std::endl;
   pcl::GeneralizedIterativeClosestPoint<pcl::PointXYZ, pcl::PointXYZ>::Ptr gicp(
     new pcl::GeneralizedIterativeClosestPoint<pcl::PointXYZ, pcl::PointXYZ>());
-  pcl::PointCloud<pcl::PointXYZ>::Ptr aligned = align(gicp, target_cloud, source_cloud);
+  align(gicp, target_cloud, source_cloud);
```

Comment:
Why remove `aligned = `?
If you're going to delete it, you should remove everything except for the final align function.
If you decide not to delete it, then for the sake of code consistency, it would be better to keep everything.

Reply:
I change it to suppress the notation from cppcheck by adding `// cppcheck-suppress redundantAssignment` at b0c4075 .

To use the inline suppression, use `--inline-suppr` args.